earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/globe_cutter_app.py

```diff
@@ -305,8 +305,7 @@ def CreateInfoFile(self):
     """Create globe info file."""
     fp = open(self.info_file, "w")
     fp.write("Portable Globe\n")
-    fp.write("Copyright 2017 Google Inc.
-              Licensed under the Apache License, Version 2.0.\n")
+    fp.write("Copyright 2017 Google Inc.\nLicensed under the Apache License, Version 2.0.\n");
```

Comment:
Does this work with the semicolon? You don't typically see it in python code. I say we should remove it for consistency